app/models/booking.rb

```diff
@@ -44,6 +44,11 @@ class Booking < ApplicationRecord
   scope :by_status, ->(status){where status: status if status.present?}
   scope :by_user_id, ->(id){where user_id: id}
   scope :order_by_status, ->{order :status}
+  scope :get_data_chart, (lambda do
+                            where status: [Settings.booking.status.accept,
+                                            Settings.booking.status.cancel]
+                          end)
+  scope :within_20_days, ->{where "updated_at >= ?", DateTime.now - 20.days}
```

Comment:
viết 1 scope để mở rộng sau này dùng nhiều chỗ nữa nhé
truyền vào time
```suggestion
  scope :updated_at_glt, ->time{where "updated_at >= ?", time}
```

Reply:
vâng chị